config.yaml.default

```diff
@@ -0,0 +1,33 @@
+project_working_dir: /tmp/mediasync
+allowed_extensions:
+ - jpg
+ - png
+operation_mode: copy
+base_path:
+driver: local
+
+mergin:
+  url: https://public.cloudmergin.com
+  username: media-sync
+  password: super-secret
+  project_name: media-sync/project
+
+local:
+  dest: /tmp/mediasync_copy
+
+minio:
+  endpoint: localhost:9000
+  access_key: EXAMPLE
+  secret_key: EXAMPLEKEY
+  bucket: test
+  secure: false
+  region:
+
+references:
```

Comment:
we should mention somewhere (readme/changelog?) that structure has been changed and migration of .ini is needed and update of env variables as well for clients who use older version